### DISCUSSION TITLE
Shared Modeling files and MOBI Files won't load properly

### DISCUSSION THREAD

**Discussion by DonkeySaddle** [Open-Systems-Pharmacology/Forum#1330] [Questions & Problems] [answered]
Whenever I try to open a "Shared Modeling File" or export a simulation from a "PK-Sim Project File" to MOBI something goes wrong and I just get this(a blank looking MOBI file): 

![Screenshot (2)](https://user-images.githubusercontent.com/25123758/194735351-f8a359f1-c49f-4ea8-b8f3-69764bbca162.png)

All the simulation data is there (e.g., if I click the simulation icon all the reactions, events, etc. from my simulation seem to have been imported), but my simulation clearly has not loaded properly and I can't run it/manipulate it. Though, I can run it in R (assuming the simulation doesn't contain the [dimension ](https://github.com/Open-Systems-Pharmacology/Forum/discussions/1322) Flow per body surface area).

I have uninstalled/reinstalled my OSPsuite, but I am still coming up short. Any idea what might be wrong? Likely I am doing something incredibly stupid...

Any help would be greatly appreciated. 

Thanks,

DS

**Comment by msevestre** [comment, marked as answer]
Hi @DonkeySaddle 
This happens for some unknown reasons when you are docking undocking a computer. The layout gets sometimes corrupted 
Fear not: There is a simple way of dealing with this

1. Close MoBi
2. Go to `c:\users\<YOUR_USER_NAME>\AppData\Roaming\Open Systems Pharmacology\MoBi\11.0` and delete the file called "setting.xml
3. Restart MoBi
4. Rejoy

**Reply by DonkeySaddle** [reply]
@msevestre This worked! WOW! There is no way I was figuring that out without you. Thanks so much.

DS

**Reply by msevestre** [reply]
We need to implement a way to do this from.the user interface. We have it in PKsim but not in mobi. 
Glad it worked
